index.js

```diff
@@ -124,6 +125,25 @@ module.exports = class Kernel {
     }
   }
 
+  createHandler (create) {
```

Comment:
wdyt about createHandler -> runCreate?

Things that are "handlers" are irreducible components for the kernel. That is they are code that have special privileges. I think should be able to get by with 5 handlers, three remaining ones don't make sense until we have shards or are running in a desktop environment. 

the "runSomething" methods are thing that can be moved to contract's at some point. Create is one of those things. The create contract does something like this

```
contract Create (msg, code){
      var caller = msg.caller
      var nonce = Storage[caller].nonce
      var newAddress = sha3(nonce, caller)      
      Storage[newAddress] = code
}
```

Now the above ignores the initialization code that contract run. To do that the code handler would have to be exposed via the interface.

```
codeHandler(initializationCode, newAddress, gas ...env_params)
```

This would run the initialization code.  The Storage Root would be newAddress, so the code running would only have access in storage to paths "below"  newAddress.